package/MDAnalysis/analysis/align.py

```diff
@@ -123,17 +126,22 @@
    >>> ref = mda.Universe(PSF, PDB_small)
    >>> mobile = mda.Universe(PSF, DCD)     # we use the first frame
    >>> align.alignto(mobile, ref, select="protein and name CA", weights="mass")
+   (21.892591663632704, 6.809396586471809)
 
 This will change *all* coordinates in *mobile* so that the protein
 C-alpha atoms are optimally superimposed (translation and rotation).
 
 To **fit a whole trajectory** to a reference structure with the
 :class:`AlignTraj` class::
 
+   >>> import doctest
+   >>> doctest.ELLIPSIS_MARKER = '-etc-'
```

Comment:
I don't think that's needed if you use the standard `...` ellipsis. Don't redefine the marker.
```suggestion
```